data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
+			choice
+				`	(Check it out.)`
+				`	(Not interested.)`
+					decline
+			`	Following the signage, you notice a substantial group of people gathered in front in front of a large warehouse that stands out amongst the rest of the warehouses in the yard. As you wait in the queue, a young woman lines up behind you with a readily apparent exuberance and asks you, "Just out of curiosity, how much do you know about Hope and its history?"`
+			choice
+				`	"Not much at all."`
+				`	"I come from the Dirt Belt, but I really don't know too much about what happened on Hope."`
+				`	"I've been there myself and know what happened, but other than that I don't know."`
+					goto "been to hope"
+					to display
+						has "visited planet: Hope"
+						or
+							not "Expedition to Hope 2: done"
+							not "FW Katya 4: done"
+				`	"I've been there myself, actually, and met some people affected by it."`
+					goto "been to hope"
+					to display
+						or
+							has "Expedition to Hope 2: done"
+							has "FW Katya 4: done"
+			`	"Well, then it's a good thing you're seeing Adelita's exhibition on Hope. You'll be surprised at how powerful it is, and you'll learn a lot about the whole sad tragedy that happened there and what Parliament and the Paradise Worlds did to everybody who tried to help," she adds somewhat sullenly. "You know, this was their first work that really got them famous in the interplanetary scene, but they've been forced to close it, so today's their final day. I had to see it one more time before they went."`
+				goto questions
+
+			label "been to hope"
+			`	"Oh, wow! It must have been a harrowing experience. Adelita does a powerful job of giving you the whole story, though, and I bet you'll still learn a lot. You might even feel as if you've been transported back there yourself," she says with a mischievous grin. "You know, this was their first work that really got them famous in the interplanetary scene, but they've been forced to close it and today's their final day. I had to see it one more time before they went."`
+
+			label questions
+			choice
+				`	"Who's Adelita?"`
+				`	"Why did they have to close it?"`
+					goto signage
+
+			`	"They're an independent journalistic collective whose goals are to bring the injustices of human civilization to light. No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their use of mixed media, their eye for story telling... I just love it. I know you're going to love it too," she says.`
+				goto exhibition
+
+			label signage
+			`	"Well, this exhibition used to be permanent, and it garnered a lot of attention, but the planetary government was pressured by the Paradise Planets to force Adelita to leave, because it gave such a candid view of what they did to those poor people. Personally, I think what Parliament and the Paradise Planets have been doing to the people of Hope and their advocates like Adelita has just been abhorrent and wrong."`
```

Comment:
```suggestion
			`	"Well, this exhibition used to be permanent, but the planetary government was pressured by the Paradise Planets to force Adelita to leave. What the Navy and the Paradise Planets did to the people of Hope was abhorrent. Now Adelita's bringing light to it, and they're being silenced. It's wrong."`
```
Same as before; I just think this sounds more like how someone would talk. Also, it's mentioned later that this was more of a Navy thing than from Parliament. Open to suggestions!

Reply:
It was the Navy that acted on their own initiative to save the people of Hope, and Parliament that punished them for doing so.

> "The eruption was on the other continent," she explains, "so no one was killed in the blast. It was a slow-moving disaster. We had six months or so to move two million settlers off-world before the temperatures would drop too low to be livable. More than half of the Navy was mobilized to ferry our people to other planets."
> "Here's the best part," says Eyes. "The Navy operates with a certain amount of autonomy. Turns out when they found out about the disaster, they mobilized without even waiting to clear it with Parliament. Which was technically within their mandate, this being a natural disaster. But the Navy is funded by taxes, and the Paradise Planets pay ten times as much in taxes as the Dirt Belt."
> "The Paradise Planets viewed the Navy as their own private security force," says Katya, "so they got really mad when half the Navy suddenly vanished to assist some dirty refugees. Because those admirals gave the order to assist us, millions of lives were saved. And then Parliament fired them."

Reply:
```suggestion
			`	"Well, this exhibition used to be permanent, but the planetary government was pressured by the Paradise Planets to force Adelita to leave. What Parliament and the Paradise Planets did to the people of Hope was abhorrent. Now Adelita's bringing light to it, and they're being silenced. It's wrong."`
```